includes/class-indieauth-token-ui.php

```diff
@@ -59,8 +59,9 @@ public function new_token() {
 		if ( empty( $scopes ) ) {
 			$scopes = 'create update';
 		}
-		$scopes = sanitize_text_field( $scopes );
-		$token  = self::generate_local_token( $client_name, $scopes );
+		$scopes  = sanitize_text_field( $scopes );
+		$expires = sanitize_text_field( $_REQUEST['expires_in'] );
```

Comment:
Can `$_REQUEST['expires_in']` be empty or unset?

Reply:
I originally was going to add the option to the authorize form, but didn't....so this line is unnecessary...but I may add that feature.